app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not ("vaccine" in visitor.keys()):
+            raise NotVaccinatedError("NotVaccinatedError")
+        if datetime.date.today() > visitor["vaccine"]["expiration_date"]:

```

Comment:
use get method instead of parenthesis everywhere to avoid KeyError
`        if datetime.date.today() > visitor.get("vaccine", {}).get("expiration_date"):
`